packages/next/src/server/stream-utils/node-web-streams-helper.ts

```diff
@@ -134,14 +116,14 @@ export function createBufferedTransformStream(): TransformStream<
   })
 }
 
-export function createInsertedHTMLStream(
+function createInsertedHTMLStream(
   getServerInsertedHTML: () => Promise<string>
 ): TransformStream<Uint8Array, Uint8Array> {
+  const encoder = new TextEncoder()
   return new TransformStream({
-    async transform(chunk, controller) {
-      const insertedHTMLChunk = encodeText(await getServerInsertedHTML())
-      controller.enqueue(insertedHTMLChunk)
-      controller.enqueue(chunk)
+    start: async (controller) => {
+      const html = await getServerInsertedHTML()
+      controller.enqueue(encoder.encode(html))
```

Comment:
This looks like a behavior change. The old code ensures that if there is new inserted chunk during the stream (after the `start` phase), it will still be enqueued. The new code changes that.

Reply:
Never mind! I see, updating :)